source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -20,29 +20,69 @@ There are a some limitations for this kind of use case:
 -  The imapc code doesn't support many IMAP features. Most importantly
    SEARCH isn't supported, which may result in lower performance.
 
+-  Sharing mailboxes in a cluster is supported only for obox and mdbox.
```

Comment:
Where did this come from? I don't see why it would make much of a difference. And if mdbox is supported, then definitely sdbox would be supported as well..

Reply:
Yeah right, i just removed this sentence now.